config/manifests/bases/clusterserviceversion.yaml.in

```diff
@@ -52,8 +52,7 @@ spec:
       deployments: null
     strategy: ""
   installModes:
-  # CSVs featuring a conversion webhook can only support the AllNamespaces install mode
-  - supported: false
+  - supported: true
```

Comment:
since we don't have conversion for any CRDs, we can support OwnNamespace installmode.

Reply:
> Is is problematic for a deployment with OwnNamespace, and we introduce conversion webhooks later on? Can that deployment move to AllNamespaces?

That's the exact problem being faced right now.

OLM has no way to automatically upgrade operator group which influences installModes
https://olm.operatorframework.io/docs/advanced-tasks/operator-scoping-with-operatorgroups/

This operator group is installed by admin/UI, will need to manually changed/ discussed with olm devs.

But for now, we don't need to put this restriction since we don't have conversion webhooks right now.